token/program-2022-test/transfer-hook-test-programs/downgrade/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+//! Program implementation
+
+use ::{
```

Comment:
nit: did you mean to include this extra `::` ?

Reply:
oops, sure didn't thanks!